[kwa-kfp-component] Introduce KWA's frontend component for kfp links

elenzio9 opened the pull request:
In this PR:
- Introduce the `kfp-run` component as a distinct component.
- Make the pipeline button a link.

Here's a screenshot:
![image](https://user-images.githubusercontent.com/87971102/199739578-0cc242b5-b57e-470f-9ebd-2488ff3f3729.png)

Signed-off-by: Elena Zioga <elena@arrikto.com>

johnugeorge commented:
Can you rebase?

kimwnasptd commented:
@elenzio9 Before merging this PR I'd suggest to first have a GH action that continuously checks the tests.

@orfeas-k can you take a stab at it?

johnugeorge commented:
@elenzio9 Can you post a screenshot of the change?

elenzio9 commented:
> @elenzio9 Can you post a screenshot of the change?

I just updated the PR description with a screenshot, but as you can see it remains the same. We introduced the `kfp-run` component as a distinct component because we'll need it for an upcoming effort where we'll re-use the common table we have for the Trials list. This way we'll be able to integrate easier with future efforts for sorting, pagination and filtering.

Related issues: https://github.com/kubeflow/katib/issues/1441, https://github.com/kubeflow/kubeflow/issues/6460, https://github.com/kubeflow/kubeflow/issues/6462

kimwnasptd commented:
@elenzio9 now that we've merged the tests PR with @orfeas-k https://github.com/kubeflow/katib/pull/1998, can you rebase on top of the latest `master` and force push?

I'd expect that
1. Only the tests relevant with the frontend will get triggered
2. The unit tests will be running

kimwnasptd commented:
Great to see that the tests pass and that they are just the ones for the frontend!

/lgtm
/approve
